### PR TITLE
Update the docs with the latest HTTP port numbers

### PR DESCRIPTION
**Pull request by digadeesh**
## 🗣 Description

Update the documentation with the latest http ports.

## 🔨 Related Issues

https://github.com/spiceai/spiceai/pull/2034

### CODE REVIEW

File: spiceaidocs/docs/clients/grafana/index.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ This tutorial creates and configures Grafana and Prometheus locally to scrape an
     docker-compose up
     ```
 
-1. Go to `http://localhost:3000/dashboard/import` and add the JSON from [monitoring/grafana-dashboard.json](https://github.com/spiceai/spiceai/blob/trunk/monitoring/grafana-dashboard.json).
+1. Go to `http://localhost:8090/dashboard/import` and add the JSON from [monitoring/grafana-dashboard.json](https://github.com/spiceai/spiceai/blob/trunk/monitoring/grafana-dashboard.json).
```
Comment (phillipleblanc) [inline spiceaidocs/docs/clients/grafana/index.md]:
```suggestion
1. Go to `http://localhost:3000/dashboard/import` and add the JSON from [monitoring/grafana-dashboard.json](https://github.com/spiceai/spiceai/blob/trunk/monitoring/grafana-dashboard.json).
```

This port is for Grafana, and should stay at 3000

File: spiceaidocs/docs/clients/grafana/index.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ pagination_prev: 'clients/index'
 pagination_next: null
 ---
 
-The Spice runtime has a Prometheus endpoint configured by default on port `9000`. There's also prebuilt Grafana dashboards for out of the box monitoring of all your Spice instances.
+The Spice runtime has a Prometheus endpoint configured by default on port `9090`. There's also prebuilt Grafana dashboards for out of the box monitoring of all your Spice instances.
```
Comment (phillipleblanc) [inline spiceaidocs/docs/clients/grafana/index.md]: Technically its not configured by default if you run `spiced` directly. It's only configured by default if you launch the runtime from the CLI via `spice run`

File: spiceaidocs/docs/components/data-connectors/odbc.md [hunk 3]
```diff
@@ -128,7 +128,7 @@ datasets:
 All together now:
 
 ```bash
-$ docker run -p3000:3000 -p50051:50051 -v $(pwd)/spicepod.yaml:/spicepod.yaml -v $(pwd)/test.db:/test.db -it spice-libsqliteodbc --http=0.0.0.0:3000 --flight=0.0.0.0:50051
+$ docker run -8090:8090 -p50051:50051 -v $(pwd)/spicepod.yaml:/spicepod.yaml -v $(pwd)/test.db:/test.db -it spice-libsqliteodbc --http=0.0.0.0:8090 --flight=0.0.0.0:50051
```
Comment (phillipleblanc) [inline spiceaidocs/docs/components/data-connectors/odbc.md]:
```suggestion
$ docker run -p8090:8090 -p50051:50051 -v $(pwd)/spicepod.yaml:/spicepod.yaml -v $(pwd)/test.db:/test.db -it spice-libsqliteodbc --http=0.0.0.0:8090 --flight=0.0.0.0:50051
```

File: spiceaidocs/docs/clients/grafana/index.md (hunk 2 above)
Comment (lukekim) [reply spiceaidocs/docs/clients/grafana/index.md]: We really need an entire dedicated monitoring section.

Comment (digadeesh) [reply spiceaidocs/docs/clients/grafana/index.md]: The document will be updated with the language similar to monitoring with Datadog for consistency.  Monitoring section can come in a separate PR.

**Review by sgrebnov — APPROVED** [review]

**Pull request merged**